index.html

```diff
@@ -0,0 +1,158 @@
+      <fieldset class="border p-2 rounded">
+        <legend class="w-auto">
+          Personal Information:
+        </legend>
+        <div class="row">
+          <div class="col-lg-8 col-sm-6">
+            <label for="last-name">Surname:</label>
+            <input type="text" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="last-name" placeholder="Фамилия" required>
+          </div>
+          <div class="col-lg-4 col-sm-6">
+            <label for="first-name">Name:</label>
+            <input type="text" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="first-name" placeholder="Имя" required>
+          </div>
+          <div class="col-lg-12 col-sm-12">
+            <label for="input-age">How old are you?:</label>
+            <input type="number" class="form-control shadow-lg p-3 mb-5 bg-white rounded" min="1" max="100" name="years" id="input-age" required>
+          </div>
+          <div class="col-lg-6 offset-lg-2 offset-sm-3 col-sm-6">
+            <label for="input-birthday">Full date of birth:</label>
+            <input type="date" name="birth" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="input-birthday">
+          </div>
+        </div>
+        <div class="col-lg-6 offset-lg-2 offset-sm-3 col-sm-6">
+          <div class="form-group form-check">
+            <input type="checkbox" class="input-form-check" id="input-check-accept" required>
+            <label class="label-form-check" for="input-check-accept">I accept the terms of the agreement</label>
+          </div>
+        </div>
+      </fieldset>
+      <br>
+      <fieldset class="border p-2 rounded">
+        <legend class="w-auto">
+          Registration
+        </legend>
+        <div class="row">
+          <div class="col-lg-6 col-sm-6">
+            <div class="form-group">
+              <label for="input-email">E-mail</label>
+              <input type="email" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="input-email" placeholder="name@example.com" required>
+            </div>
+          </div>
+          <div class="col-lg-6 col-sm-6">
+            <div class="form-group">
+              <label for="input-password">Password</label>
+              <input type="password" class="form-control shadow-lg p-3 mb-5 bg-white rounded" id="input-password" placeholder="Password" required>
+            </div>
+          </div>
+        </div>
+      </fieldset>
+      <br>
+      <fieldset class="border p-2 rounded">
+        <legend class="w-auto">
+          An interesting fact about you!
+        </legend>
+        <div class="row">
+          <br>
+          <div class="col-lg-4 col-sm-6">
+            <div class="form-check">
+              <legend class="col-form-label">Do you love cats?</legend>
+              <input class="form-check-input" type="radio" name="exampleRadios" id="input-check-yes" value="option1" checked>
```

Comment:
Use names consistently.
You are using kebab-case everywhere in the file, and it's only `exampleRadois` written in camelCase.